### PR TITLE
chore: optimization high contrast header removal

### PR DESCRIPTION
**Pull request by at-susie**
### Description
Implement design optimizations for high contrast header removal from VR.

wizard-optimization

Related links, issue #, if available: n/a

### How has this been tested?

<details>
   <summary>Review checklist</summary>

_The following items are to be evaluated by the author(s) and the reviewer(s)._

#### Correctness

- _Changes include appropriate documentation updates._
- _Changes are backward-compatible if not indicated, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#public-apis)._
- _Changes do not include unsupported browser features, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#browsers-support)._
- _Changes were manually tested for accessibility, see [accessibility guidelines](https://cloudscape.design/foundation/core-principles/accessibility/)._

#### Security

- _If the code handles URLs: all URLs are validated through [the `checkSafeUrl` function](https://github.com/cloudscape-design/components/blob/main/src/internal/utils/check-safe-url.ts)._

#### Testing

- _Changes are covered with new/existing unit tests?_
- _Changes are covered with new/existing integration tests?_
</details>

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/internal/utils/content-header-utils.ts [hunk 1]
```diff
@@ -5,3 +5,5 @@ import { getGlobalFlag } from './global-flags';
 export const contentHeaderClassName: string = getGlobalFlag('removeHighContrastHeader')
   ? ''
   : 'awsui-context-content-header';
+
+export const isHighContrastHeaderActive = !!getGlobalFlag('removeHighContrastHeader');
```
Comment (michaeldowseza) [inline src/internal/utils/content-header-utils.ts:9]: Can we rename this to `shouldRemoveHighContrastHeader`?

Comment (just-boris) [reply src/internal/utils/content-header-utils.ts:9]: Can we also have a single `getGlobalFlag` call?

```
export const isHighContrastHeaderActive = !!getGlobalFlag('removeHighContrastHeader');

export const contentHeaderClassName = isHighContrastHeaderActive ? '' : 'awsui-context-content-header';
```

Comment (michaeldowseza) [reply src/internal/utils/content-header-utils.ts:9]:
```
export const shouldRemoveHighContrastHeader = !!getGlobalFlag('removeHighContrastHeader');
export const contentHeaderClassName = shouldRemoveHighContrastHeader ? '' : 'awsui-context-content-header';
```

**Comment by at-susie** [comment]
Thank you @michaeldowseza @just-boris for the review. 
I updated the `getGlobalFlag` call accordingly.

**Review by just-boris — APPROVED** [review]

**Pull request merged**